### Academic Experience
###### Bachelor of Science (BS),&nbsp;&nbsp; Department of Biophysics, Panjab University, Chandigarh, India- 2002-2005- Biophysics 
###### Doctor of Philosophy (PhD), School of health sciences, Swinburne University of Technology, Victoria, Australia- 2008-2011- Biophysics
###### Post-doctoral training, Beth Israel Deaconess Medical Center/Harvard University, Boston, USA- 2011-2016- Cancer genomics and bioinformatics 


### Professional Experience

##### 2020- Present:
###### Assistant Professor, Sidney Kimmel Cancer Center, Thomas Jeffersons University and Hospital, Philadelphia, PA
###### Job Responsibilities:
<ul type="circle">
    <li> As a part of my research, I develop algorithms and methods for high-throughput genomic data analysis. </li>
    <li> Along with the research, I perform high-throughput sequencing data analysis for various investigators inside and outside of Sidney Kimmel Cancer Center. I mainly perform whole genome and exome analysis, RNA-seq differential expression analysis, RNA-seq variant calling, single cell RNA-seq data analysis. </li>
    <li> Provide training to the researchers to analyze the high-throughput genomic data. </li>
</ul>

<br>
<br>

##### 2018- 2019:
###### *Computational Biologist,* *__Sidney Kimmel Cancer Center, Thomas Jeffersons University and Hospital, Philadelphia, PA__*
###### Job Responsibilities:
<ul type="circle">
    <li> As a part of my research, I develop algorithms and methods for high-throughput genomic data analysis. </li>
    <li> Along with the research, I perform high-throughput sequencing data analysis for various investigators inside and outside of Sidney Kimmel Cancer Center. I mainly perform whole genome and exome analysis, RNA-seq differential expression analysis, RNA-seq variant calling, single cell RNA-seq data analysis. </li>
    <li> Provide training to the researchers to analyze the high-throughput genomic data. </li>
</ul>

<br>
<br>

##### 2017- 2018:
###### *Assistant Bioinformatics Scientist,* *__Emory Integrated Computational Core, Emory University, Atlanta, GA__*
###### Job Responsibilities:
<ul type="circle">
    <li> I developed and maintained various pipelines for genomic data analysis using R, python, and Unix language. </li>
    <li> Also, developed, designed and deployed pipelines on AWS EC2 instances, creating AMIs and data storage on S3. </li>
    <li> Performed high-throughput sequencing data analysis- whole genome/exome, RNA-seq, ChIP-seq, ATAC-seq, 16s rRNA, microarrays for various investigators of Emory University. </li>
    <li> Taught 'genomic data analysis' segment of bioinformatics course (IBS574) to graduate student.</li>
</ul>

<br>
<br>

##### 2016- 2017:
###### *Bioinformatics Analyst,* *__Emory Integrated Computational Core, Emory University, Atlanta, GA__*
###### Job Responsibilities:
<ul type="circle">
    <li> I developed and maintained various pipelines for genomic data analysis using R, python, and Unix language. </li>
    <li> Performed high-throughput sequencing data analysis- whole genome/exome, RNA-seq, ChIP-seq, ATAC-seq, 16s rRNA, microarrays for various investigators of Emory University. </li>
    <li> Taught 'python programming' segment of bioinformatics course (IBS574) to graduate student.</li>
</ul>

<br>
<br>

##### 2011- 2016:
###### *Research fellow,* *__Beth Israel Deaconess Medical Center, Harvard University and The Broad Institute, Boston, MA__*
###### Job Responsibilities:
<ul type="circle">
    <li> Performed microarray data analysis for the minimally invasive cancer therapy projects. </li>
    <li> Also, data mine the exisiting literature to find any novel biomarkers in minimally invasive cancer therapy field. </li>
    <li> Developed pipelines for RNA-seq data analysis using python programming language.</li>
</ul>